models/datasets/reservoir/ethereum/tokens.sql

```diff
@@ -1,6 +1,6 @@
 {{ config(    
     schema = 'reservoir',
-    alias = 'tokens_beta',
+    alias = 'tokens',
```

Comment:
Maybe rename the old one to avoid collision?

Reply:
the old one shouldn't be in use, was just for testing